extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -109,6 +120,12 @@ public void build(CombinedIndexBuildItem indexBuildItem, BuildProducer<Reflectiv
         // classes needed to perform reflection on DirectByteBuffer - only really needed for Java 8
         reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, "java.nio.DirectByteBuffer"));
         reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, "sun.misc.Cleaner"));
+
+        // enable native JAAS/SASL PLAIN support
+        reflectiveClass
```

Comment:
I would add the LoginModule directly here.

Reply:
Actually, I need all the sub-classes, for example (but not limited to):
* PlainLoginModule (org.apache.kafka.common.security.plain)

The list of sub-classes also contains:
* ExternalCertificateLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* Krb5LoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* PropertiesLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* DigestLoginModule (org.apache.zookeeper.server.auth)
* LDAPLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* ScramLoginModule (org.apache.kafka.common.security.scram)
* NTLoginModule (com.sun.security.auth.module) 
* GuestLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* InVMLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* FileLoginModule (com.sun.jmx.remote.security)
* JndiLoginModule (com.sun.security.auth.module)
* AbstractKeycloakLoginModule (org.keycloak.adapters.jaas)
* LdapLoginModule (com.sun.security.auth.module)
* OAuthBearerLoginModule (org.apache.kafka.common.security.oauthbearer)
* UnixLoginModule (com.sun.security.auth.module)
* Krb5LoginModule (com.sun.security.auth.module)
* CertificateLoginModule (org.apache.activemq.artemis.spi.core.security.jaas)
* KeyStoreLoginModule (com.sun.security.auth.module)

Reply:
Relocated native-SASL/JAAS code into separate build step.